projects/ccf-rui/src/app/shared/components/rotation-slider/rotation-slider.component.ts

```diff
@@ -45,6 +45,27 @@ export class RotationSliderComponent {
    */
   constructor(private readonly ga: GoogleAnalyticsService) { }
 
+  ngOnInit() {
+    const inputX = document.querySelector('.slider-x');
```

Comment:
Inject `ElementRef<Element>` and use `ref.nativeElement.querySelector(...)`
This will make sure you don't accidentally grab anything outside this element. Especially when there are multiple instances of this component